### PR TITLE
[TESTING RECOMMENDED] Various perk-related changes and perk buff/rework/tweak/i dont know anymore

### PR DESCRIPTION
**Pull request by Kegdo**
## About The Pull Request

legit crying rn writing this changelog was torture

merged some really weak perks, buffed others, new images and names for various perks

Removed Toxic Revenger perk

Added Triumph of the Will perk - increase positive breakdown chance by 30
images in comments

Perks merged: (perks : merge_product)
Lungs of Iron, Blood of Lead, Balls of Plasteel: Prime Endurance
Fast Fingers, Market Professional: Profit Maker
Gunslinger, Gunsmith: Gunmaster
Quiet as Mouse, Nightcrawler: Nightcrawler
Horrible Deeds, Terrible Fate: Menace to Society
Ear of Silver merged with Rat fate

Perks buffed:
Moralist: when people around restore sanity, gain insight; healing people by bandaging or ointmenting(lmao) restores sanity a bit, still take sanity dmg around people with < 45 sanity
Drug addict: addiction is now permanent, but you gain a bottle of stims instead of just drugs
Alcoholic: addiction is now permanent, but you increase all combat stats by 15
Rat: merged with Ear of Silver perk
Unfinished delivery: restore blood, increase heal, sleep time reduced to 15, 100% chance(still fair i swear~~, fuck RNG~~)
Space Asshole: no bomb res, but can dive into disposal chutes and gain no damage from disposals, slightly increased fall res
Charming Personality: Changed to Mentor: Increase insight gain multiplier, increase sanity gain multiplier, people around recover sanity
Ass of Concrete: if you dive or get thrown into an unreinforced window and it breaks, move on the tile where it was, increase damage dealt to structures(more info below in related changes)
Absolute grab: fixed the lunge proc that didn't even work, increased its range by **1.**
Sure Step: Never trip while running on underplating
Paper Worm: chance increase is now 40(was 20)

Related changes:
Added stuff from #6976  and #6978 - related to space_asshole and ass of concrete
Added more info to the character setup fate menu for readability/accessibility
The withdrawal procs are human_only now and decrease your sanity gradually
probably something else?
## Why It's Good For The Game
Some perks right now are literally useless or are focused on one certain thing, or just fucking BROKEN(looking at you, absolute grab); this pr makes them all ~~overpowered~~ useful in their own extent - the focus was to make them provide you a new ability that is not beneficial in direct combat. Exceptions are absolute grab(not for everyone), prime endurance, unfinished delivery and alcoholic, but you still aren't really strong if you have them. There are also some related QoL/patch changes.
Also the Menace to Society perk name is really funny and I don't know why.
Also new images cool.
Also Toxic Revenger is very dumb not cool and i dislike it, salt pr.
Also withdrawal buff.
## Changelog
:cl:
add: new perk - Triumph of the Will
del: Toxic Revenger perk
fix: Bugged absolute grab proc
fix: prevents players with the weakened status from being able to speed up movement by diving repeatedly instead of crawling. By hex37
balance: Titanfall removed by Humon, cancel him on twitter
balance: diving into windows and doors takes longer to break them, damages the jumper - by Humon
balance: rebalanced a lot of perks
imageadd: new various perk images
tweak: the withdrawal procs are human_only now and decrease your sanity gradually
tweak: probably something else
/:cl:

### CODE REVIEW

**Comment by Kegdo** [comment]
how do i make long screenshots

**Comment by Sparrowbird611** [comment]
>del: Toxic Revenger perk

Absolutely unplayable. Jk Praise god.

**Comment by Kegdo** [comment]
oh also the withdrawal procs are human_only now and decrease your sanity gradually

**Comment by FortuneMaster** [comment]
no merge untill toxic revenger removal reversal
prime endurance may be too much with balls of plasteel
profit maker - cant see logic behind
ear of silver - could probably leave as a standalone perk(so possible to get outside of rat)
drug addict - is just slightly less shit
alcoholic - isnt it basically a better freelancer now?

**Comment by Sparrowbird611** [comment]
Bruh, reversing toxic revenger removal? Play the game more then we'll talk. The real problem here is absolute grab having 2x the grab speed instead of its lunge.

**Comment by Kegdo** [comment]
> no merge untill toxic revenger removal reversal

Lol.
>drug addict - is just slightly less shit 

Lmao.
> alcoholic - isnt it basically a better freelancer now?

No. **Permanent addiction, huge sanity debuffs on withdrawal**.

> The real problem here is absolute grab having 2x the grab speed instead of its lunge.

correct, I'll think of a better concept, but no fixing lunge

**Comment by Kegdo** [comment]
> Iike alcohol is hard to come buy or dangerous for health

it is, it gives you debuffs and makes you drunk, unless you are really robust

> Perk is hell of a fun for my toxic ass.

image
**Play the game**

**Comment by FortuneMaster** [comment]
> > Iike alcohol is hard to come buy or dangerous for health
> 
>> it is, it gives you debuffs and makes you drunk, unless you are really robust
> 
>like you should get yourself drunk
> > Perk is hell of a fun for my toxic ass.
> 
>> image 
>>**Play the game**
> I played, perk is fun)

**Comment by Sparrowbird611** [comment]
Ok hear me out.

Aside from your retarded ass fortune, nobody thinks it's fun. Ergo, delete. Cope and seethe otherwise.

**Comment by Voldirs** [comment]
*FARTS*

**Comment by ValoTheValo** [comment]
> no merge untill toxic revenger removal reversal

cope, seethe, mald. toxic revenger was absolutely awful and nobody would use it as a perk. ive seen people discard good oddities because they had toxic revenger

**Comment by Jan-Blasiak** [comment]
> no merge untill toxic revenger removal reversal

It's an awful perk and you should feel bad for liking it. I hope you eat an apple with a worm inside.

**Comment by TheShown911** [comment]
Toxic revenge great perk you should make it twice, no three times as strong as you are buffing all the other perks.

**Comment by ValoTheValo** [comment]
> Toxic revenge great perk you should make it twice, no three times as strong as you are buffing all the other perks.

make toxic revenger also do oxygen, brute, burn and radiation damage

**Comment by Kegdo** [comment]
heard your comments, will make toxic revenger gib every mob on the z-level including src

**Comment by Kegdo** [comment]
> correct, I'll think of a better concept, but no fixing lunge

i lied. get trol'd

**Comment by Sparrowbird611** [comment]
So what are you doing then.

> > correct, I'll think of a better concept, but no fixing lunge
> 
> i lied. get trol'd

**Comment by thedarkcrusader** [comment]
> > no merge untill toxic revenger removal reversal
> 
> Perk is hell of a fun for my toxic ass.
> 
> > No. Permanent addiction, huge sanity debuffs on withdrawal.
> 
> Iike alcohol is hard to come buy or dangerous for health

What???????

**Comment by Kegdo** [comment]
> So what are you doing then.
> 
> > > correct, I'll think of a better concept, but no fixing lunge
> > 
> > 
> > i lied. get trol'd

i fixed lunge.

**Comment by FortuneMaster** [comment]
> > no merge untill toxic revenger removal reversal
> 
> It's an awful perk and you should feel bad for liking it. I hope you eat an apple with a worm inside.

Eating an apple would be alraedy bad enought for me

File: code/datums/perks/fate.dm [hunk 1]
```diff
@@ -60,56 +61,57 @@
...
+			Carry this fire with you - light the way for others."
 
 /datum/perk/fate/drug_addict
 	name = "Drug Addict"
-	icon_state = "medicine" //https://game-icons.net/1x1/delapouite/medicines.html
+	icon_state = "addict" //https://game-icons.net/1x1/delapouite/medicines.html
 	desc = "For reasons you cannot remember, you decided to resort to major drug use. You have lost the battle, and now you suffer the consequences. \
-			You start with an addiction to a random drug, as well as a bottle of pills containing the drug."
+			You start with a permanent addiction to a random stimulator, as well as a bottle of pills containing the drug. \
+			Beware, if you get addicted to another stim, you will not get rid of the addiction."
```
Comment (Humonitarian) [inline code/datums/perks/fate.dm:71]:
```suggestion
			Beware, if you get addicted to another stimulant, you will not get rid of the addiction."
```

File: code/datums/perks/oddity.dm [hunk 2]
```diff
@@ -5,60 +5,56 @@
...
-
-/datum/perk/oddity/terrible_fate
-	name = "Terrible Fate"
-	desc = "You realize the painful truth of death. You don't want to die and despise death - dying is a unmistakable horror to you. \
-			Anyone who is around you at the moment of your death must roll a Vigilance sanity check. If they fail, their sanity will instantly be dropped to 0."
-	icon_state = "murder" // https://game-icons.net/1x1/delapouite/chalk-outline-murder.html
+	icon_state = "fast" // https://game-icons.net
+
+/datum/perk/oddity/gunmaster
+	name = "Gunmaster"
```
Comment (Humonitarian) [inline code/datums/perks/oddity.dm:11]: Name sounds really bland

Comment (Kegdo) [reply code/datums/perks/oddity.dm:11]: then suggest something better :mitosis:

Comment (Voldirs) [reply code/datums/perks/oddity.dm:11]: Gun Nut?

Comment (Kegdo) [reply code/datums/perks/oddity.dm:11]: how about i gun your n-

Comment (Firefox13) [reply code/datums/perks/oddity.dm:11]: Gun Done

**Comment by Firefox13** [comment]
> Some perks right now are literally useless or are focused on one certain thing

that's the fucking point of them

**Review by Firefox13 — COMMENTED** [review]
I do not like the perk merges - perks are indeed made to be special and specific in their own way and "focused on one certain thing".

**Comment by Kegdo** [comment]
> > Some perks right now are literally useless or are focused on one certain thing
> 
> that's the fucking point of them

being useless is not a point; being focused on one certain thing is a point, but i had a brain fart.

**Review by Firefox13 — COMMENTED** [review]
besides you swapped around the icons but not the links to them and this is a copyright violation (probably)

**Comment by Kegdo** [comment]
> besides you swapped around the icons but not the links to them and this is a copyright violation (probably)

our codespeak icons has the 'codespeak' name, it's fine if that one works

**Comment by Kegdo** [comment]
> I do not like the perk merges - perks are indeed made to be special and specific in their own way and "focused on one certain thing".

omg that 33% chance to heal my organs by 20 points and put me to sleep for 20 or so seconds is so good op and fun to play!! i sure like trying to find oddities in trash instead of doing something else like robbing guild!! its so fun because i get to just not get any good perk!! omg that 50% damage resistance against oxygen damage is so useful to me because 95% of eris threats are burn and brute damge!!!! :shock_emoji:
omg that toxin revenger perk surely is fun for everyone and not just shit!!

**Comment by Kegdo** [comment]
**__Omg that market professional perk is SOOOO OP!!! its not like it can be completely replaced by a one(1) single device that everyone who might need that perk already has!__**

**Comment by drexample** [comment]
what is this

**Comment by Holvr0** [comment]
Why remove toxic revenger?

Much better idea would be to make it so a person with this perk doesn't just passively stink, but rather they "vent" their toxloss.

so if they have no tox, they're absolutely normal. But if they have any tox, they vent a % of it each tick, possibly spreading it to people nearby.

**Comment by Jan-Blasiak** [comment]
> Why remove toxic revenger?
> 
> Much better idea would be to make it so a person with this perk doesn't just passively stink, but rather they "vent" their toxloss.
> 
> so if they have no tox, they're absolutely normal. But if they have any tox, they vent a % of it each tick, possibly spreading it to people nearby.

Now that's better.

**Comment by drexample** [comment]
image

**Comment by drexample** [comment]
Fixes are nice, though, put them in another PR.

**Comment by Kegdo** [comment]
bruh.

**Comment by Kegdo** [comment]
> image

how about you actually tell me what's the problem with the pr so i can make it better, instead of just telling me to go fuck myself and closing the whole thing?

**Pull request closed without merging**